trunk/NDHMS/Land_models/NoahMP/phys/module_sf_noahmpdrv.F

```diff
@@ -1212,6 +1214,122 @@ SUBROUTINE TRANSFER_MP_PARAMETERS(VEGTYPE,SOILTYPE,SLOPETYPE,SOILCOLOR,CROPTYPE,
 
  END SUBROUTINE TRANSFER_MP_PARAMETERS
 
+SUBROUTINE PEDOTRANSFER_SR2006(nsoil,sand,clay,orgm,parameters)
```

Comment:
SR2006? Maybe drop in what that is. A comment with the citation.

Reply:
Saxton and Rawls (2006)

Reply:
@jmccreight I do have the reference in the option definition. Since this is an attempt to unify with external codes, I would prefer to hold off on an addition like this until after the release and/or when we make this part of the code active.